ingestion/functions/parsing/common/python/parsing_lib/input_event.json

```diff
@@ -7,7 +7,7 @@
     "uploadIds": ["123456789012345678901234"],
     "dateFilter": {
         "numDaysBeforeToday": 3,
-        "op": "EQ"
+        "op": "LT"
```

Comment:
somehow this was needed to pass unit tests because that file is read by the e2e test, I have no idea how this passed in prod as it failed continuously locally.

Reply:
I suspect there's some timezone goofiness that impacts this; I don't remember in which TZ our actions run.